bindings/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "bindings"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+elements = "0.25.0"
+thiserror = "1.0.50"
+uniffi = { version = "0.29.2", features = ["cli", "tokio"] }
+serde_json = "1"
+boltz-client = { path = "..", features = ["ws", "esplora", "electrum"] }
+bitcoin = "0.32.5"
+async-trait = "0.1.88"
+tokio = "1.44.1"
```

Comment:
Let's inherit shared dependencies from the workspace. Like this: https://github.com/BoltzExchange/boltz-backend/blob/master/boltzr-cli/Cargo.toml#L9